src/pages/Delivery/Delivery.styles.js

```diff
@@ -0,0 +1,7 @@
+export const styleDelivery = {
+  display: "flex",
+  flexDirection: "column",
+  margin: "3rem auto",
+  width: "1152px",
+  height: "1251px",
+}
```

Comment:
please don't forget to add the new line 
<img width="432" alt="Screenshot 2022-11-18 at 10 22 24 p m" src="https://user-images.githubusercontent.com/3700448/202833876-c7f5653f-23ec-4a3e-9e90-3e962e9e4138.png">